rtm/stack.py

```diff
@@ -94,11 +97,12 @@ def get_peak_coordinates(S, global_max=True, height=None, min_time=None,
         peak_dt = (S.time.data[1] - S.time.data[0]) / np.timedelta64(1, 's')
 
         # Find all peaks based on set thresholds
-        peaks, props = find_peaks(s_peak, height, distance=min_time/peak_dt)
+        peaks, props = find_peaks(s_peak, height, distance=min_time/peak_dt,
+                                  prominence=prominence)
 
         npeaks = len(peaks)
-        print(f'Found {npeaks} peaks in stack for height > {height:.1f} and '
-              f'min_time > {min_time:.1f} s.')
+        print(f'Found {npeaks} peaks in stack for height > {height:.1f}, '
+              f'min_time > {min_time:.1f} s, and prominence > {prominence}.')
```

Comment:
Should this be printed if `prominence=None`? I think maybe no.

Reply:
I agree that it should not be printed.

Reply:
OK updated @liamtoney. Let me know if you think ready.